### ISSUE TITLE
Request: Varnish 6 LTS packages for Debian Bullseye

### ISSUE THREAD

**Issue by janosmiko** [varnishcache/pkg-varnish-cache#150]
Hello,

Is it possible to add support for Varnish6 LTS built on Debian Bullseye?

``` bash
docker run --rm -it debian:bullseye bash
root@b74c9a5ce0cf:/# apt-get update && apt-get install -y curl
root@b74c9a5ce0cf:/# curl -sSL https://packagecloud.io/install/repositories/varnishcache/varnish60lts/script.deb.sh | bash
Detected operating system as debian/11.
...
# supressed
...

root@b74c9a5ce0cf:/# cat /etc/apt/sources.list.d/varnishcache_varnish60lts.list
# this file was generated by packagecloud.io for
# the repository at https://packagecloud.io/varnishcache/varnish60lts

deb https://packagecloud.io/varnishcache/varnish60lts/debian/ bullseye main
deb-src https://packagecloud.io/varnishcache/varnish60lts/debian/ bullseye main

root@b74c9a5ce0cf:/# apt-get update
Hit:1 http://deb.debian.org/debian bullseye InRelease
Hit:2 http://security.debian.org/debian-security bullseye-security InRelease
Hit:3 http://deb.debian.org/debian bullseye-updates InRelease
Ign:4 https://packagecloud.io/varnishcache/varnish60lts/debian bullseye InRelease
Err:5 https://packagecloud.io/varnishcache/varnish60lts/debian bullseye Release
  404  Not Found [IP: 52.8.129.94 443]
Reading package lists... Done
E: The repository 'https://packagecloud.io/varnishcache/varnish60lts/debian bullseye Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.


# the only varnish package is the default from debian
root@b74c9a5ce0cf:/# apt-cache show varnish
Package: varnish
Version: 6.5.1-1
Installed-Size: 2717
Maintainer: Varnish Package Maintainers <team+varnish-team@tracker.debian.org>
Architecture: amd64
Provides: varnishabi-12.0, varnishabi-strict-1dae23376bb5ea7a6b8e9e4b9ed95cdc9469fb64
Depends: libc6 (>= 2.29), libedit2 (>= 2.11-20080614-0), libjemalloc2 (>= 2.1.1), libncursesw6 (>= 6), libpcre3, libtinfo6 (>= 6), libvarnishapi2 (>= 6.5.0), adduser, gcc, libc6-dev | libc6.1-dev | libc-dev, lsb-base
Pre-Depends: init-system-helpers (>= 1.54~)
Suggests: varnish-doc
Description: state of the art, high-performance web accelerator
Description-md5: ebbadfc1a0485abd58ba563250510a25
Homepage: https://www.varnish-cache.org/
Tag: implemented-in::c, interface::daemon, network::server, protocol::http,
 role::program, uitoolkit::ncurses, use::proxying
Section: web
Priority: optional
Filename: pool/main/v/varnish/varnish_6.5.1-1_amd64.deb
Size: 957564
MD5sum: 081c6da73c69cfae97800aac1adf6463
SHA256: 837c9e9a464d4d5f473cbd54455dfe8c47cf26eecfdb1218b878d7b46297309e

**Comment by janosmiko** [comment]
I've just found that this same issue exist for Varnish 6.6 as well and even if I'm using the official Varnish Docker Image and trying to install varnish-dev to the image.

```
docker run --rm -it varnish:6.6 bash
root@841bddff8cff:/etc/varnish# apt-get update -yq && apt-get install curl -yq
...

root@841bddff8cff:/etc/varnish# curl -sSL https://packagecloud.io/install/repositories/varnishcache/varnish66/script.deb.sh | bash
Detected operating system as debian/11.
...
...
The repository is setup! You can now install packages.
root@841bddff8cff:/etc/varnish#
root@841bddff8cff:/etc/varnish# apt-cache search varnish-dev
libvarnishapi-dev - development files for Varnish
```

**Comment by gquintard** [comment]
Hi,

This should be useful: https://varnish-cache.org/lists/pipermail/varnish-misc/2020-August/026926.html

TL;DR:
- `bullseye` support is in `master` already so `7.0` will get it (released this week if all goes well)
- earlier, non-LTS version won't be backported to it
- `6.0LTS` will possibly get it

If you want to help with the last point, you can open a PR to update the `6.0` branch here: https://github.com/varnishcache/varnish-cache/blob/6.0/.circleci/config.yml#L274, incorporating the new distribution. Once we have a new release, the packages will be created.

**Comment by janosmiko** [comment]
Thank you @gquintard !

**Comment by gquintard** [comment]
No problem. I'll try to make the Soviet bro in the email a bit more
visible, you aren't the first one to ask about that

**Comment by martin-g** [comment]
> Soviet bro in the email

Is this me ?! :-)
I am from Bulgaria, but we are close enough (geographically and mentally :-) )

**Comment by gquintard** [comment]
Oooof, that was one bad case of of autocorrect!
